### ISSUE TITLE
Foam darts can embed and kill

### ISSUE THREAD

**Issue by stoneman2** [Bubberstation/Bubberstation#420]
## Round ID: 
1905

## Testmerges:
#208, #294, #248, #241, #314, #367, #369, #371, #306, #382

## Reproduction:
Shoot someone with a foam gun. If it hits, it will embed in their limbs, causing them to bleed.

**Comment by Serxule** [comment]
not a bug, you can put nails inside darts, which is a feature

**Comment by stoneman2** [comment]
No, this is even without nails. I tested this with on a local server and it still happens.
![image](https://github.com/Bubberstation/Bubberstation/assets/34432122/9ed9f454-9078-474d-9020-04a15009655e)
Perhaps only with SMG, testing more.

**Comment by Serxule** [comment]
oh, yeah thats probably a bug then if its actually doing damage/wound, i assume it mightve been given embed (to mimic the dart sticking since its a suction cup dart) but something got fucked up

**Comment by stoneman2** [comment]
From testing, I can confirm. It only happens with the SMG. Dart safety's on. Every other gun works fine, no damage. Only SMG that lodges without fail.

**Comment by Serxule** [comment]
weird

**Comment by Majkl-J** [comment]
Fixed with https://github.com/tgstation/tgstation/pull/77450
